documentation/modules/cruise-control/proc-deploying-cruise-control.adoc

```diff
@@ -9,23 +9,13 @@ To deploy Cruise Control to your Strimzi cluster, define the configuration using
 
 Deploy one instance of Cruise Control per Kafka cluster.
 
-NOTE: Step one is only required if automatic topic creation is disabled in the Kafka cluster.
-
 .Prerequisites
 
 * A Kubernetes cluster
 * A running Cluster Operator
-* You are aware of the guidance on xref:#auto-created-topics-logs[Auto-created topics and log compaction].
 
 .Procedure
 
-. If automatic topic creation is disabled (`auto.create.topics.enable: false` is configured in `spec.kafka.config`), first create the `strimzi.cruisecontrol.metrics` topic:
-+
-[source,shell,subs="attributes+"]
-----
-kubectl apply -f examples/cruise-control/cruise-control-topic.yaml
-----
-
 . Edit the `cruiseControl` property of the `Kafka` resource.
 +
 The properties you can configure are shown in this example configuration:
```

Comment:
Following the new and less strict documentation rules ... maybe you can remove the proceudure structure (The prerequisites heading, the procedure heading etc.) which seem to have lot more noise than value.

Reply:
> Quite a lot of changes ... is this PR still about the CC upgrade to 2.0.108 only? Or should we give it a better name?

There are a lot of changes, but they are all due to the switch to the new version.

Reply:
I think @laidan6000 has said he will do a follow up PR which will add the CC metric topic info etc to the intro so maybe these edits could be made in that PR?